services/gateway/apis/http-version3/swagger/definitions/validators.json

```
@@ -21,6 +25,11 @@
                         "format": "id",
                         "example": "3c95f7931d61909ff092375fc8ad2bc35e393b62d5cca902",
                         "description": "BLS key"
```

Please add the `.` at the end of the descriptions for `generatorKey` as well.
```suggestion
                        "description": "BLS key."
```